src/test/crimson/seastar_runner.h

```diff
@@ -71,16 +71,28 @@ struct SeastarRunner {
     auto ret = app.run(argc, argv, [this] {
       on_end.reset(new seastar::readable_eventfd);
       return seastar::now().then([this] {
-	begin_signaled = true;
-	[[maybe_unused]] auto r = ::eventfd_write(begin_fd.get(), APP_RUNNING);
-	assert(r == 0);
-	return seastar::now();
+// FIXME: The stall detector uses glibc backtrace function to
+// collect backtraces, this causes ASAN failures on ARM.
+// For now we just extend timeout duration to 10000h in order to
+// get the same effect as disabling the stall detector which is not provided by seastar.
+// the ticket about migrating to libunwind: https://github.com/scylladb/seastar/issues/1878
+// Will remove once the ticket fixed.
+#ifdef __aarch64__
+        seastar::smp::invoke_on_all([] {
+          using namespace std::chrono;
+          seastar::engine().update_blocked_reactor_notify_ms(duration_cast<milliseconds>(10000h));
+        }).get();
+#endif
+        begin_signaled = true;
+        [[maybe_unused]] auto r = ::eventfd_write(begin_fd.get(), APP_RUNNING);
+        assert(r == 0);
+        return seastar::now();
```

Comment:
nit: ditto, whitespaces - cleaner diff would make it easier to understand/revert in the future

Reply:
ditto